lang/compiler.go

```diff
@@ -132,7 +133,7 @@ func CompileTableObject(to *flux.TableObject, now time.Time, opts ...CompileOpti
 
 func WalkIR(astPkg *ast.Package, f func(o *flux.Operation) error) error {
 
-	if spec, err := spec.FromAST(context.Background(), astPkg, time.Now()); err != nil {
+	if spec, err := spec.FromAST(context.Background(), nil, astPkg, time.Now()); err != nil {
```

Comment:
We we need deps here? How is WalkIR used? I am not familiar with it. Can you add a doc comment to the function once you figure it out?

Reply:
it's used in BucketsAccessed in InfluxDB.   This will not require deps, our plan is to have the `Default` deps be the always-error case that we discussed offline.   This will be done as a followup.  i'd rather not have the deps be nil anywhere to prevent panics.